attest/verifier/src/dcap.rs

```diff
@@ -55,14 +58,26 @@ impl DcapVerifier {
     }
 
     /// Verify the `evidence`
-    pub fn verify(
-        &self,
-        evidence: &Evidence<Vec<u8>>,
-    ) -> VerificationOutput<AndOutput<EvidenceValue, AndOutput<ReportData, Attributes>>> {
+    pub fn verify(&self, evidence: &Evidence<Vec<u8>>) -> VerificationOutput<DcapVerifierOutput> {
         self.verifier.verify(evidence)
     }
 }
 
+impl VerificationMessage<DcapVerifierOutput> for DcapVerifier {
+    fn fmt_padded(
+        &self,
+        f: &mut Formatter<'_>,
+        pad: usize,
+        result: &VerificationOutput<DcapVerifierOutput>,
+    ) -> core::fmt::Result {
+        let is_success = result.is_success();
+        let status = choice_to_status_message(is_success);
+
+        writeln!(f, "{:pad$}{status} DCAP evidence:", "")?;
```

Comment:
I don't have a strong preference on what this header line should be, so up for suggestions on making it clearer if needed.